app-ios/Sources/TimetableFeature/TimetableView.swift

```diff
@@ -63,8 +63,6 @@ public let timetableReducer = Reducer<TimetableState, TimetableAction, Timetable
     case let .selectDay(day):
         state.selectedDay = day
         return .init(value: .refresh)
-    case .selectItem:
-        return .none
```

Comment:
Maybe removing this case gets the build error.

Reply:
Exactly. 

@surajsau 
Please revert this change 🙏🏼

Reply:
Done and fixed 👍